docs/tubemq_user_guide.md

```diff
@@ -1,4 +1,29 @@
 # TubeMQ User Guide
+## Prerequisites
+
+- Java 7/8(Java 9+ are not test)
+- Maven
+- [protoc 2.5.0](https://github.com/protocolbuffers/protobuf/releases/tag/v2.5.0)
+
+## Install protoc
```

Comment:
I suggest removing this ## Install protoc section. One concern is the instructions are only for Windows; another concern is protoc is just prerequisites as others. People should be able to find how to install protoc through Internet. That should be enough we list it as an item of prerequistes.

Reply:
It's a good suggestion, I will remove